taxi/views.py

```diff
@@ -1,47 +1,54 @@
+from django.contrib.auth.decorators import login_required
```

Comment:
The import statement for `login_required` should be on a separate line to follow PEP 8 guidelines. Consider separating it from the `LoginRequiredMixin` import.